logger/logger.go

```diff
@@ -183,20 +185,21 @@ func newLogWriter(logType string) io.Writer {
 	if rotationHook != "" {
 		options = append(options, rotatelogs.WithHandler(NewRotateHandler(rotationHook)))
 	}
-
 	var w io.Writer
+	var t time.Duration
 	if rotateEnable {
 		switch rotationTime {
 		case "hourly":
 			logSuffix = ".%Y%m%d%H"
+			t = 1 * time.Hour
 		case "daily":
 			logSuffix = ".%Y%m%d"
-		case "monthly":
```

Comment:
Since time.Duration makes it difficult for me to specify a month, I erased it because I think that it is not used.